readme.md

```diff
@@ -120,7 +120,7 @@ The programmer comes home with 12 loaves of bread.
 
 **Q:** Why was the computer cold?
 
-**A:** It left its Windows open!
+**A:** It left it's Windows open!
```

Comment:
"its" is the neutral-gender third-person possessive pronoun. "it's" is an abbreviation of "it is".

Reply:
PR compromised, just revert the whole thing

Reply:
Want to submit a new PR?